spec/model/dirty_spec.rb

```diff
@@ -5,19 +5,31 @@
   context "checking dirty attributes" do
     before do
       Her::API.setup url: "https://api.example.com" do |builder|
+        user1 = { id: 1, fullname: "Lindsay Fünke" }
+        user2 = { id: 2, fullname: "Maeby Fünke" }
+        post1 = { id: 1, user_id: 1, body: "Hello" }
+
         builder.use Her::Middleware::FirstLevelParseJSON
         builder.use Faraday::Request::UrlEncoded
         builder.adapter :test do |stub|
-          stub.get("/users/1") { [200, {}, { id: 1, fullname: "Lindsay Fünke" }.to_json] }
-          stub.get("/users/2") { [200, {}, { id: 2, fullname: "Maeby Fünke" }.to_json] }
+          stub.get("/users")   { [200, {}, [user1, user2].to_json] }
```

Comment:
Can we stick w/ the current, explicit style to keep the specs consistent? Also think it helps to see the json w/ each stub 😺